src/Select.jsx

```diff
@@ -58,7 +58,7 @@ const Select = React.createClass({
       });
       let onclick = null;
       if (!item.disabled) {
-        onclick = this.onSelect.bind(this, +item.value);
+        onclick = this.onSelect.bind(this, item.value);
```

Comment:
It was explicitly converted by number, but that's not what I need handling 'AM/PM' selector